services/frontend-service/pkg/cmd/server.go

```diff
@@ -542,6 +564,24 @@ func (p *Auth) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getUserFromDex(w http.ResponseWriter, req *http.Request, clientID, baseURL, DexRbacPolicyPath string) *auth.DexAuthContext {
+	policy, err := auth.ReadRbacPolicy(c.DexEnabled, c.DexRbacPolicyPath)
```

Comment:
we already did `auth.ReadRbacPolicy` during startup. We shouldn't have to do it again here.

Reply:
Agreed!